db/migrate/20170302134438_fix_only_superseded_editions.rb

```diff
@@ -1,16 +1,20 @@
 class FixOnlySupersededEditions < ActiveRecord::Migration[5.0]
   def up
-    Document
+    docs = Document
       .joins(:editions)
       .where(%q(
         NOT EXISTS (
           SELECT id FROM editions AS e
           WHERE e.state IN ('draft', 'published', 'unpublished')
             AND e.document_id = documents.id
         )
-      )).find_each do |document|
-        document.editions.order(updated_at: :desc).first
-          .update_column(:state, "unpublished")
-      end
+      ))
+
+    docs.find_each do |document|
+      document.editions.order(updated_at: :desc).first
+        .update_column(:state, "unpublished")
+    end
+
+    Commands::V2::RepresentDownstream.new.(docs.map(&:content_id))
```

Comment:
This will probably go wrong as the transaction is not committed at the time this is called. Is it also a pain point running locally?